### PR TITLE
Create documentation

### PR DESCRIPTION
**Pull request by BattleCh1cken**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+## Table of Contents
+
+- [I Have a Question](#i-have-a-question)
+- [I Want To Contribute](#i-want-to-contribute)
+- [Reporting Bugs](#reporting-bugs)
+- [Suggesting Enhancements](#suggesting-enhancements)
+- [Your First Code Contribution](#your-first-code-contribution)
+- [Improving The Documentation](#improving-the-documentation)
+- [Styleguides](#style-guides)
```
Comment (BestUsernamEver) [inline .github/CONTRIBUTING.md:25]: Style guide instead of styleguide?

File: docs.typ [hunk 2]
```diff
@@ -0,0 +1,387 @@
+#import "./docs-template.typ": *
+
+#import "@preview/tidy:0.1.0"
+
+#let version = toml("/typst.toml").package.version
+#let import-statement = "#import \"@preview/tidy:" + version + "\""
+
+#show: docs-template.with(
+  title: "The Notebookinator", subtitle: "Easy formatting for robotics notebooks.", abstract: [
+    Welcome to the Notebookinator, a Typst package meant to simply the notebooking
```
Comment (Isaiah15414) [inline docs.typ:10]: **Simplify** instead of simply

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (Isaiah15414) [inline .github/CONTRIBUTING.md:25]: **Style Guides** instead of styleguides

**Review by Isaiah15414 — APPROVED** [review]

File: docs.typ (hunk 2 above)
Comment (BattleCh1cken) [reply docs.typ:10]: Did you ever apply this change?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (BattleCh1cken) [reply .github/CONTRIBUTING.md:25]: Did you apply this change before merging?

File: docs.typ (hunk 2 above)
Comment (Isaiah15414) [reply docs.typ:10]: No I didn't know I could change it, I've never done a pull request before so I thought you would see this and change it, sorry.

**Pull request merged**